src/common/styles/root-level-only/global-styles.scss

```diff
@@ -65,3 +65,13 @@ button::after {
         color: $spinner-text;
     }
 }
+
+// Add hover state that's not yet in FluentUI to menu items:
```

Comment:
The code for this change looks good as a workaround for the fluent UI issue, but can this comment link the corresponding fluent UI issue so we can track when we can remove the workaround?